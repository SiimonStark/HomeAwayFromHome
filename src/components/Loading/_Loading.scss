.Loading {
  height: 90%;
  width: 80%;
  position: relative;
  z-index: 5;

    img {
      border: ridge white 3px;
      border-radius: 435px;
      height: 55%;
      width: 60%;
    }
}

.standby {
  margin: none;
  font-family: $font-Three;
}