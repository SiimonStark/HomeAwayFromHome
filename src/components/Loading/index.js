import React from 'react';

const Loading = () => {
  return (
    <article className="Loading">
      <h2>Loading:</h2>
      <p className="standby">Please Standby</p>
      <img src='https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/d9b42947064155.586ee48b83acf.gif' alt="Loading gif" />
    </article>
  )
}

export default Loading;