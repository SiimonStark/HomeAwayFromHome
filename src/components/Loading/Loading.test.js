import React from 'react';
import { shallow } from 'enzyme';
import Loading from '.';

describe('Loading', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Loading />)
  })
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})