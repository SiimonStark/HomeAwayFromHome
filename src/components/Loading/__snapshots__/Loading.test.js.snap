// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot 1`] = `
<article
  className="Loading"
>
  <h2>
    Loading:
  </h2>
  <p
    className="standby"
  >
    Please Standby
  </p>
  <img
    alt="Loading gif"
    src="https://mir-s3-cdn-cf.behance.net/project_modules/max_1200/d9b42947064155.586ee48b83acf.gif"
  />
</article>
`;
